zkvm/entrypoint/Cargo.toml

```diff
@@ -13,7 +13,7 @@ once_cell = "1.8.0"
 rand = "0.8.5"
 serde = { version = "1.0.196", features = ["derive"] }
 libm = { version = "0.2.8", optional = true }
-sha2 = "0.10.8"
+sha2 = { git = "https://github.com/succinctbot/RustCrypto-hashes.git" }
```

Comment:
this should be `https://github.com/sp1-patches/RustCrypto-hashes` with branch `patch-v0.10.8`